### ISSUE TITLE
Ensure spreadsheet tracker can answer all the questions in for KR1

### ISSUE THREAD

**Issue by mshadbolt** [ebi-ait/hca-ebi-wrangler-central#11]
**Description of the task:**

As wranglers we have been asked to make a list of suitable datasets for the DCP2 MVP. As we already have many datasets present in the spreadsheet tracker we need to make sure we can answer all the questions asked by running reports/pivots/filtering on the tracker. Ideally this would be done before adding more data to the tracker.

Questions are specified in #6 

**Acceptance criteria for the task:**

- [ ] reports can be generated to answer all the questions in #6

**Comment by mshadbolt** [comment]
@zperova and @mshadbolt also agreed to be available for feedback.

Other wranglers to provide ideas

**Comment by ami-day** [comment]
First draft located here: https://docs.google.com/spreadsheets/d/1lJFPR-loZuBhKdijOcSOn7wqe_-N4MYWKJkLFfRqefU/edit#gid=165291923

I need to review the brokering folder tomorrow to double check I haven't missed any datasets I converted from GEO previously.

@mshadbolt @zperova feedback very welcome, would you mind reviewing?

**Comment by zperova** [comment]
@ami-day yes HCA publication list is now ready, please use HCA publication filter in the Dataset Tracker

**Comment by ami-day** [comment]
Updated version can be found here: https://docs.google.com/spreadsheets/d/1DfRFfZpZEmWU6aubKwIZ4D0lfm038IW0fmHRigS8yBU/edit#gid=552473761

@rays22 please could you add your ArrayExpress tracking info. to this version?

**Comment by rays22** [comment]
> @rays22 would you mind adding the info. you collect about ArrayExpress datasets here?

@ami-day 
I have added a few new datasets to the table [Datasets for DCP2.0 MVP](https://docs.google.com/spreadsheets/d/1lJFPR-loZuBhKdijOcSOn7wqe_-N4MYWKJkLFfRqefU/). <https://docs.google.com/spreadsheets/d/1lJFPR-loZuBhKdijOcSOn7wqe_-N4MYWKJkLFfRqefU/>
I was not sure into which section to put the datasets that have been archived and have passed HCA spreadsheet metadata validation. I think they are closer to `Complete` than to `Maybe`, so I added them to the `Complete` section.

I am also going to copy this comment to <https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/6>

**Comment by mshadbolt** [comment]
I made a query sheet within the original tracker workbook because I think this is more sustainable going forward - https://docs.google.com/spreadsheets/d/1rm5NZQjE-9rZ2YmK_HwjW-LgvFTTLs7Q6MzHbhPftRE/edit#gid=1830648942

Any row in the tracker that has a status in the MVP_status column will be added to this query sheet. I currently have the statuses 'included' and 'maybe' to cover the two sheets present in the additional sheets that were created.

I added the columns 'project location' and 'fastq location' to the tracker so that you can see where data/metadata is. 

Nothing in this new query sheet can be edited, the data must be updated in the tracker sheet and any edits and updates are automatically populated to the new sheet. 

Similarly any filtering, sorting or addition of columns needs to be done by editing the formula in A2 of this sheet, no one should be able to edit any other cell. 

Happy to take people through how it works if they are interested. Also open to suggestions for additional columns/information that should be added or anything that is missing from this view that was in the additional spreadsheet.

**Comment by ami-day** [comment]
I think this is considered done now - @lauraclarke can we close the issue?
